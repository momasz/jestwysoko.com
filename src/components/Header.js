import React from "react";

const Header = () => (
  <h1><a href="/">jestwysoko.com</a></h1>
);

export default Header;
