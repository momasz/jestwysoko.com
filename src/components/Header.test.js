import React from 'react';
import { mount } from 'enzyme';
import Header from './Header';

describe('Header component', () => {
  it('renders as expected', () => {
    const header = mount(
      <Header/>
    );
    expect(header).toMatchSnapshot();
  });
});
