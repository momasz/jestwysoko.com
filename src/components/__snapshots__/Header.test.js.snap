// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders as expected 1`] = `
<Header>
  <h1>
    <a
      href="/"
    >
      jestwysoko.com
    </a>
  </h1>
</Header>
`;
